gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/ValueMapTest.java

```diff
@@ -125,6 +131,30 @@ public void g_VX1X_outXcreatedX_valueMap() {
 
     }
 
+    /**
+     * TINKERPOP-1483
+     * 
+     */
+    @Test
+    @LoadGraphWith(MODERN)
+    public void valueMapHasObjectKeys() {
```

Comment:
This method is named wrong, it should be like the traversal generator method name, minus the `get_` prefix.